### ISSUE TITLE
version `GLIBC_2.34' not found

### ISSUE THREAD

**Issue by anarion80** [svenstaro/miniserve#1110]
Latest docker image keeps restarting with the following error:

`/app/miniserve: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /app/miniserve)`

**Comment by fcastilloec** [comment]
This error was introduced on `v0.23.1`, I tested `v0.23.0` and there are no issues with that version.

**Comment by anarion80** [comment]
I'm not sure. I just run `docker.io/svenstaro/miniserve:0.23.1@sha256:6611541b419c6835f61c5d9806aa9b42cea40f984d48e8e3c5a6b1347035456a` and it is fine, no error.
`v0.23.1` was pushed to Docker 6 days ago.
Latest was pushed 2 days ago. Switching to `latest` immediately results in restarts.

**Comment by fcastilloec** [comment]
@anarion80 you are right. I mistakenly assumed that `latest` was also pointing to the last release and wasn't a different version. `v0.23.1` doesn't show any errors either, the problem was introduced afterward.

**Comment by svenstaro** [comment]
Thanks guys. The current CI environment uses a more recent version of glibc than the distroless/cc image. This causes the observed problem. One way to solve that is to switch to a different image with a more recent version of glibc. Sadly that will cause the image sizes to blow up quite a bit. I'm now switching to `debian:testing-slim` which seems like a fair compromise until more recent distroless images are available.

**Comment by svenstaro** [comment]
New `latest` images should be fixed now.

**Comment by woss** [comment]
this is happening with the `0.23.2`
![image](https://github.com/svenstaro/miniserve/assets/80440/18eb236e-b578-4d63-86bf-f58cc9c02fd6)
![image](https://github.com/svenstaro/miniserve/assets/80440/1dd8fe3b-8db0-4ea8-badc-1abae7389037)

**Comment by svenstaro** [comment]
Which image did you use?

**Comment by svenstaro** [comment]
Ok, if you need to use an older Linux and can't use containers, I suggest using the musl build instead which shouldn't give you this trouble.

**Comment by woss** [comment]
> Which image did you use?

I installed it from the crates using the `binstall` on vscode dev container  `mcr.microsoft.com/devcontainers/rust:0-1-bullseye` and native Debian 11

**Comment by micsama** [comment]
> Ok, if you need to use an older Linux and can't use containers, I suggest using the musl build instead which shouldn't give you this trouble.

thank you ,it works
